app/services/proofing/socure/id_plus/request.rb

```diff
@@ -3,42 +3,42 @@
 module Proofing
   module Socure
     module IdPlus
-      class RequestError < StandardError
-        def initialize(wrapped)
-          @wrapped = wrapped
-          super(build_message)
-        end
+      class Request
+        class Error < StandardError
```

Comment:
There were some autoloading issues with `RequestError` not being inside `Request`, so I moved it inside and renamed it to just `Error`.